### DISCUSSION TITLE
Kong on ppc64le

### DISCUSSION THREAD

**Discussion by licove** [Kong/kong#9609] [Help]
Hello all,

is it possible to compile and run Kong on ppc64le?

**Comment by hanshuebner** [comment]
> is it possible to compile and run Kong on ppc64le?

It is difficult to reply "no" to this as if you put enough effort into trying, you may be able to get it to run.  ppc64le is not a platform supported by Kong itself, so you'll basically have to deal with any problems that you may encounter on your own.  The first thing you'll need to take care of is getting LuaJIT to run.  There were [some efforts in porting it](https://github.com/PPC64/LuaJIT/wiki#current-status-of-the-ppc64-port-branch), but those seem to have stalled 5 years ago.

If your real question is "Is Kong supported on ppc64le", then the answer is a clear no.

**Comment by StarlightIbuki** [comment]
It cannot be done by simply running a build script. As far as I know, there were attempts to make the build tool work with ppc64le (try searching `ppc64le kong`). And there are no promises that all the dependencies could be built on ppc64le.
I believe that with some manual intervention and modification to the build process this could be done. However, even if you want to do so, you should wait for now as we are refactoring the build tool and it may be easier after the refactoring.
